website/docs/cloud-docs/api-docs/team-tokens.mdx

```diff
@@ -38,6 +38,8 @@ description: >-
 
 # Team Token API
 
+Team API tokens allow access to the workspaces that the team has access to, without being tied to any specific user. A user can create, delete and list tokens in the organization.
```

Comment:
I saw this was resolved, but I want to push back here because the current version is a bit misleading, hence my suggestion. 

`A user can create, delete and list tokens in the organization.` is correct, but not really what this page is about, so I suggested `You can create and delete team tokens and list an organization's team tokens.` because separating the two thoughts is more true to the API endpoints on this page. If you disagree, please let me know!